test/mock.ts

```diff
@@ -21,5 +23,27 @@ export class MockClass {
 }
 
 export class Call {
-  constructor(public parameters: object, public result: string | null | void){}
+  constructor(public parameters: any[], public result: string | null | void){}
+}
+
+/**
+ * Tests whether a call queue has one Call and returns it.
+ * @param mock the MockClass
+ * @param methodName the call queue's method name
+ */
+export function expectCall(mock: MockClass, methodName: string): Call {
+  expect(mock.callQueues[methodName].length).to.eq(1);
```

Comment:
Could this test be to expect that the length is greater than zero or to pass an optional parameter to set the expected length, with -1 meaning don't check? That way we can use expectCall in different situations other than a clean, one-call test.

Reply:
Providing the -1 no check option is difficult since the expectCall was intended to always return a value.  So there's an optional now if there is an expected length; else, it assume it's more than 0.  If you want to check that there were truly no calls, I added an `expectNoCalls` function.